CSharpAdvanced/GenericHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+
+public static class GenericHelper
+{
+    public static T GetFirstItem<T>(List<T> items)
+    {
+        if (items == null || items.Count == 0)
+        {
+            throw new ArgumentException("List of elements is empty.");
+        }
+
+        return items[0];
```

Comment:
it is better to use FisrtOrDefault instead of [0]
this way we can get rid of hardcoded value :)